flepimop/gempyor_pkg/src/gempyor/parameters.py

```diff
@@ -148,12 +225,28 @@ def parameters_quick_draw(self, n_days: int, nsubpops: int) -> ndarray:
 
     def parameters_load(self, param_df: pd.DataFrame, n_days: int, nsubpops: int) -> ndarray:
         """
-        drop-in equivalent to param_quick_draw() that take a file as written parameter_write()
-        :param fname:
-        :param n_days:
-        :param nsubpops:
-        :param extension:
-        :return: array of shape (nparam, n_days, nsubpops) with all parameters for all nodes and all time.
+        Format all parameters as a numpy array including sampling and overrides.
+        
+        This method serves largely the same purpose as the `parameters_quick_draw`, but
+        has the ability to override the parameter specifications contained by this class
+        with a given dataframe.
+        
+        Args:
+            param_df: A DataFrame containing the columns 'parameter' and 'value'. If 
+                more than one entry for a given parameter is given then only the first 
+                value will be taken.
+            n_days: The number of days to generate an array for.
+            nsubpops: The number of subpopulations to generate an array for.
+        
+        Returns:
+            A numpy array of size (`npar`, `n_days`, `nsubpops`) where `npar` 
+            corresponds to the `npar` attribute of this class.
+            
+        Note:
+            If any of the parameters are 'timeseries' type parameters and are not being 
+            overridden then `n_days` and `nsubpops` must be equal to the number of days 
+            between `ti` and `tf` given when initializing this class and the number of 
+            subpopulations given to this class via `subpop_names`.
```

Comment:
Similar to other comment - doesn't seem to actually be enforced / checked?

Reply:
https://github.com/HopkinsIDD/flepiMoP/pull/277#discussion_r1704046819